web/libs/editor/tests/e2e/tests/regression-tests/image-zoom-position.test.js

```diff
@@ -182,6 +182,9 @@ Data(relativeCoordsFF).Scenario('Zoomed image should keep center image in center
   for (const [shiftX, steps] of [[-200, 25], [200, 25], [-200, 1], [200, 1]]) {
     await AtOutlinerPanel.dragResizerBy(shiftX, 0, AtOutlinerPanel.resizeRight, steps);
   }
+
+  I.wait(1);
```

Comment:
Is this necessary, also what changed to require an additional wait to previous?

Reply:
This task was failing intermittently, likely because the stage had not been updated at the moment we were retrieving the stage's center position. By adding I.wait(1), the issue stopped occurring.